spec/javascripts/packages/document-capture/components/acuant-capture-spec.jsx

```diff
@@ -21,6 +22,16 @@ describe('document-capture/components/acuant-capture', () => {
     delete window.AcuantCameraUI;
   });
 
+  function initializeAcuant({ isSuccess, isCameraSupported }) {
```

Comment:
this cleans up the file so much! 👍  WDYT about calling it something like `stubAcuant`?

Reply:
> this cleans up the file so much! 👍 WDYT about calling it something like `stubAcuant`?

#4063 also implemented this, and since it's used across multiple test files, moved it to a support file with a slightly different usage as a "test lifecycle hook".

Re: "stub" vs. "initialize", I'm conflicted because technically this is doing both of those things 😄  Splitting it to separate functions could help differentiate, but it's mostly redundant, since the real-world scenario this is intending to mimic is one where Acuant script loads, assigns globals, and calls the initialization function immediately.

I think I still lean more toward "initialize", since from the developer's perspective, I care more about this expressed intent to imitate the loading behavior of the Acuant script, and only secondarily that it makes the global functions available as stubs that I can attach to.